frontend/src/components/Button/index.tsx

```diff
@@ -30,6 +30,7 @@ export const Button: React.FC<ButtonProps> = ({
 }) => {
 	const handleClick: React.MouseEventHandler<HTMLButtonElement> = (e) => {
 		if (loading) {
+			e.preventDefault()
```

Comment:
Unrelated, but a source of bugs where you could trigger the `onClick` handler multiple times even while the button was in the loading state.